mne/annotations.py

```diff
@@ -46,8 +46,6 @@ def _check_o_d_s(onset, duration, description):
     if description.ndim != 1:
         raise ValueError('Description must be a one dimensional array, '
                          'got %d.' % (description.ndim,))
-    if any([';' in desc for desc in description]):
```

Comment:
Instead of removing these lines I guess we should change them to `if any('{COLON}' in desc for desc in description)`, then ideally also add a tiny test:
```
with pytest.raises(ValueError, match='in descriptions not supported'):
    Annotations([0], [1], 'a{COLON}b')
```
or so